test/topology_custom/test_commitlog_segment_data_resurrection.py

```diff
@@ -52,78 +53,80 @@ async def get_segments_num():
     def get_cl_segments():
         return {os.path.basename(s) for s in glob.glob(os.path.join(cl_path, "CommitLog-*"))}
 
-    await cql.run_async("create keyspace ks1 with replication = {'class': 'SimpleStrategy', 'replication_factor': 1}")
-    await cql.run_async("create keyspace ks2 with replication = {'class': 'SimpleStrategy', 'replication_factor': 1}")
-    await cql.run_async("create table ks1.tbl1 (pk int, ck int, primary key(pk, ck))")
-    await cql.run_async("create table ks2.tbl2 (pk int, ck int, v text, primary key(pk, ck)) WITH gc_grace_seconds = 0")
+    async with new_test_keyspace(manager, "with replication = {'class': 'SimpleStrategy', 'replication_factor': 1}") as ks1:
+      async with new_test_keyspace(manager, "with replication = {'class': 'SimpleStrategy', 'replication_factor': 1}") as ks2:
```

Comment:
I think there is a syntax to add multiple context managers into a single `with`:
```python
with context_manager1() as cm1, context_manager2() as cm2:
    pass
```

Reply:
Hmm, okay, will try it.

Reply:
Done in [065b490](https://github.com/scylladb/scylladb/commit/065b49033c05c7e6322846ba2dd837dbfcfb0797)